### ISSUE TITLE
Intro paragraph about testing conflicts overly strong, not necessary.

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag-act#67]
This paragraph is overly strong and not necessary, and undermines the credibility of WCAG.

> "Accessibility experts often disagree on how accessibility requirements should be tested. These disagreements lead to conflicting results of accessibility tests. This is true for both manual accessibility tests as well as for automated testing done through accessibility test tools (ATTs).":

Change to:

>"This framework is intended to provide a central point where accessibility experts can agree on how accessibility requirements should be tested so as to avoid conflicting results of accessibility tests. It is intended for both manual accessibility tests as well as for automated testing done through accessibility test tools (ATTs)."

**Comment by WilcoFiers** [comment]
@DavidMacDonald, we are looking at the following change, would this address your concerns?

> This framework is intended to provide a consistent interpretation of how to test for accessibility requirements so as to avoid conflicting results of accessibility tests. It is intended for both manual accessibility tests as well as for automated testing done through accessibility test tools (ATTs).

**Comment by DavidMacDonald** [comment]
yes
